Services/Seed.cs

```diff
@@ -147,51 +147,30 @@ public class Seed : ISeed
 
             foreach (var group in template.Groups)
             {
-                try
-                {
-                    await this.storage.UpdateDeviceGroupAsync(group.Id, group, "*");
-                }
-                catch (Exception ex)
-                {
-                    this.log.Error($"Failed to seed default group {group.DisplayName}", () => new { group, ex.Message });
-                }
+                await this.storage.UpdateDeviceGroupAsync(group.Id, group, "*");
             }
 
             foreach (var rule in template.Rules)
             {
-                try
-                {
-                    await this.telemetryClient.UpdateRuleAsync(rule, "*");
-                }
-                catch (Exception ex)
-                {
-                    this.log.Error($"Failed to seed default rule {rule.Description}", () => new { rule, ex.Message });
-                }
+                await this.telemetryClient.UpdateRuleAsync(rule, "*");
             }
 
```

Comment:
Please add a log statement here for "Checking for whether simulation seed data already exists."

Reply:
Ditto -- Is the "*" required here for the ETag?

Reply:
Fixed